src/plugins/vis_type_timeseries/server/routes/post_vis_schema.ts

```diff
@@ -130,8 +131,8 @@ const seriesItems = Joi.object({
   aggregate_by: stringOptionalNullable,
   aggregate_function: stringOptionalNullable,
   axis_position: stringRequired,
-  axis_max: stringOptionalNullable,
-  axis_min: stringOptionalNullable,
+  axis_max: stringOrNumberOptionalNullable,
+  axis_min: stringOrNumberOptionalNullable,
```

Comment:
I'm seeing that we are still using a mix of both types within our own code, for example: `seriesGroup.axis_min = seriesGroup.axis_min || 0;`